Updated as of Bans, Circa 14-03-19

Monohell opened the pull request.

Monohell committed:
Updated as of Bans, Circa 14-03-19

cmeister2 commented on README.md:
```diff
@@ -10,15 +10,17 @@ You are only permitted to have *one* Commander. Commanders with Partner may be u
+_Correct as of 2019-03-14_ 
 
 - [Standard Commander ban list](https://scryfall.com/search?q=banned%3Aedh&order=name&as=grid&unique=cards)
+- Commanders with Keyword: "Partner" (You may play one of the partner commanders, but you may not have both in the command zone)
```
Isn't this covered by line 9?

Monohell replied:
I'd use this in favour of Line 9, or change Line 9 to reflect this. One of the instances is redundant, but additional clarity can't hurt.

cmeister2 replied:
In that case I'd change this line to say `Multiple Commanders with Keyword: "Partner"` to better clarify that you can still run Commanders with Partner, just not two of them.

Monohell replied:
- Multiple Commanders with the Keyword "Partner" You may still play cards with the "Partner" Keyword, however only one card with the "Partner" Keyword may be in your command zone

Proposed Change above, or is this just worse?

cmeister2 replied:
Seems better, yeah!

cmeister2 commented on README.md:
```diff
@@ -10,15 +10,17 @@ You are only permitted to have *one* Commander. Commanders with Partner may be u
 - [Demonic Tutor](https://scryfall.com/card/lea/104/demonic-tutor)
 - [Empty the Warrens](https://scryfall.com/card/dds/15/empty-the-warrens)
 - [Frantic Search](https://scryfall.com/card/uma/57/frantic-search)
+- [Library of Alexandria](https://scryfall.com/card/vma/303/library-of-alexandria)
```
Library of Alexandria is already in the standard Commander ban list and as such does not need explicitly calling out.

Monohell replied:
Fair shout, Didn't see it on my prior perusal.

Monohell committed:
Reflecting discussed changes

Updated to reflect comments raised in request
Changed: Wording to the multiple commanders comment; also removed line 9 referencing commander restriction (for duplication purposes). 
Removed: Library of Alexandria as it's on the current commander ban list

cmeister2 committed:
Minor tweak to text
